src/components/Tags/Tags.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div v-if="tags.length > 0">
+      <Tag v-for="tag in tags"
+           :key="tag"
+           :icon="icon"
+           :disabled="disabled"
+           :outlined="outlined"
+           :hovered="hovered"
+           :elevated="elevated"
+           :flat="flat"
+           :color="color"
+           class="mr-1"
+      >{{ tag }}</Tag>
+    </div>
```

Comment:
I think this is would be a good place to use `gap` rather than `margin`. As is we'll always end up with margin on the last tag which I don't think we want. If an implementation needs right margin it should be added at the implementation level. 

I'm thinking something like this

```html
<div class="tags">
  <Tag v-for....>
</div>
```

```css
.tag {
  display: flex;
  align-items: center;
  gap: var(m--1);
}
```